andela/hc-rogue-one
# Ft landing page rework 156577854

#### What does this PR do?
Create content on the landing page UI/UX to make users aware that they can create both Scheduled Tasks as well as Health Check Alerts using the system.
#### Description of Tasks to be complete
GIVEN users want to know what HealthChecks offers
when they access the landing page, then they should be aware of and get access to all features on the application.

#### How should this be manually tested?
1: Install the project and install the requirements.txt
2: Start the server by running; python manage.py runserver, open localhost  http://localhost:8080 in the browser
3: The landing page/home page is opened
4: You should be able to see the neccessary information displayed on the landing page.
#### What are the relevant pivotal tracker stories?
#156577854
#### Checklist: I completed these to help reviewers

No? Unsure? -> configure your IDE, format the code and add the changes with git add . && git commit --amend
existing code that was well tested you do not have to add tests)

No? -> write tests and add them to this commit git add . && git commit --amend


No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.


No? Unsure? -> execute command git pull --rebase origin develop
